tools/run_tests/xds_k8s_test_driver/framework/xds_k8s_testcase.py

```diff
@@ -480,13 +480,29 @@ def initKubernetesClientRunner(self) -> KubernetesClientRunner:
 
     def tearDown(self):
         logger.info('----- TestMethod %s teardown -----', self.id())
+        # Todo(xuanwn): Add same check to XdsUrlMapTestCase.
+        logger.debug('Getting pods restart times')
+        client_restarts: int = self.client_runner.get_pod_restarts(
+            self.client_runner.deployment)
+        server_restarts: int = self.server_runner.get_pod_restarts(
+            self.server_runner.deployment)
```

Comment:
Just realized this could raise an error too. Let's wrap it into a try and catch `except (retryers.RetryError, k8s.NotFound) as e:`

Reply:
And just log the error.